tests/008_stack_trace.ts

```diff
@@ -1,7 +0,0 @@
-import { throwsError } from "./subdir/mod1.ts";
-
-function foo() {
-  throwsError();
-}
-
-foo();
```

Comment:
Hmm - this one seems like a better test now that I look at them... maybe instead of deleting this rename it to error2.ts and add an out file?